regex/rvalidation/dateValidation.go

```diff
@@ -0,0 +1,14 @@
+package rvalidation
+
+import (
+	"regexp"
+)
+
+// ValidateDate validates date strings of format dd/mm/yyyy.
+func ValidateDate(hash string) bool {
+	pattern := `(^(((0[1-9]|1[0-9]|2[0-8])[\/](0[1-9]|1[012]))|((29|30|31)[\/](0[13578]|1[02]))|((29|30)[\/](0[4,6,9]|11)))[\/](19|[2-9][0-9])\d\d$)|(^29[\/]02[\/](19|[2-9][0-9])(00|04|08|12|16|20|24|28|32|36|40|44|48|52|56|60|64|68|72|76|80|84|88|92|96)$)`
```

Comment:
@souvikmaji if possible can you split pattern in sub patterns , it will be easy to understand for others
ex.  pattern = datePattern + MonthPattern + YearPattern